ECR and GCR access guides (#110)
run-x/opta-docs: content/en/tutorials/gcr_access.md

---
title: "GCR Access"
linkTitle: "GCR Access"
date: 2021-07-21
draft: false
description: Guide on how to authorize image pulling from GCR
---

> With Opta, on is not forced to use the Opta-provisioned GCR repository, nor is the Opta-provisioned GCR repository

Reviewer: same type: on -> one

forbidden from being used elsewhere. The following guide explains how.

## Understanding GCR Access

> It's GCS access, that's basically it. GCR is simply a (powerful) convenience layer around object storage in a GCS

Reviewer: ```suggestion
It's same as GCS basically. GCR is simply a (powerful) convenience layer around object storage in a GCS
```
